core/modules/layout/css/layout.admin.css

```diff
@@ -134,25 +134,24 @@
   width: 100%;
   min-height: 70px;
   margin: 0 0 15px;
-  padding: 8px;
+  padding: 8px 8px 16px 8px;
   border: 1px dashed #CCC;
   background: white;
   -webkit-box-sizing: border-box;
   box-sizing: border-box;
 }
 .layout-editor-region-title .label {
   float: left;
-  margin: 0 12px 0 0; /* LTR */
-  padding: 4px 0;
+  margin: 0 ;
+  padding: 4px 0.5em;
```

Comment:
Is there a reason to do px on one side and EM on the other?

Theoretically this could get weird if a user changed their default font size, even though that's not the most common case.